docs/config.md

```diff
@@ -227,6 +227,27 @@ cpu_only_composing_models: <comma-delimited-string-list>
 # Enables the searching of request rate (instead of concurrency)
 [ request_rate_search_enable: <bool> | default: false]
 
+# Minimum percentage of the search space to profile when using Optuna
+[ min_percentage_of_search_space: <int> | default: 5]
+
+# Maximum percentage of the search space to profile when using Optuna
+[ max_percentage_of_search_space: <int> | default: 10]
+
+# Minimum number of trials to profile when using Optuna
+[ optuna_min_trials: <int> | default: 20]
+
+# Maximum number of trials to profile when using Optuna
+[ optuna_max_trials: <int> | default: 200]
+
+# Number of trials without improvement before triggering early exit when using Optuna
+[ optuna_early_exit_threshold: <int> | default: 10]
+
+# Use the concurrency formula instead of searching the concurrency space in Optuna search mode
```

Comment:
@[tgerdesnv](https://github.com/tgerdesnv) tgerdesnv [30 minutes ago](https://github.com/triton-inference-server/model_analyzer/pull/894#discussion_r1629499542)
Can the formula be specified or linked here?

Reply:
I've specified the formula